tutor/src/models/course/builder-ux.js

```diff
@@ -178,7 +178,7 @@ export default class CourseBuilderUX extends BaseModel {
 
   // skips
   skip_new_or_copy() {
-    return !!this.source || Courses.nonPreview.isEmpty;
+    return !!this.source || Courses.isEmpty || Courses.nonPreview.isEmpty;
   }
```

Comment:
hm, wouldn't `nonPreview` be empty if courses also empty? But if this fixes the bug?